### PR TITLE
check linkage

### PR DESCRIPTION
**Pull request by elharo**
Interestingly the check fails.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -296,9 +296,16 @@
...
+          <dependency>
+            <groupId>com.google.cloud.tools</groupId>
+            <artifactId>linkage-checker-enforcer-rules</artifactId>
+            <version>0.1.1</version>
+          </dependency>
+        </dependencies>
         <executions>
           <execution>
-            <id>enforce</id>
+            <id>convergence</id>
```
Comment (loosebazooka) [inline pom.xml]: Is this execution necessarily only `convergence`?

Comment (elharo) [reply pom.xml]: This is just a name. It doesn't really mean anything, but they're can't be two rules named "enforce" or Maven complains.

Comment (loosebazooka) [reply pom.xml]: Right, but I don't know what the default behavior is. Shouldn't this remain just `enforce`. Why did we change it?

Comment (elharo) [reply pom.xml]: It should never have been enforce. That's just copy-pasta from the docs.

**Pull request merged**